server/src/services/desks.service.ts

```diff
@@ -53,6 +59,59 @@ export function getDesksService({
   ];
 }
 
+export async function getDesksServiceReal({
+  zoneId,
+  deskIds = [],
+  count = 10,
+  unit = "days",
+}: GetDesksServiceParams): Promise<DeskDto[]> {
+  const desksResult: DeskDto[] = [];
+
+  if (Array.isArray(deskIds) && deskIds.length > 0) {
+    for (const deskId of deskIds) {
+      let stats = await getStats(zoneId, deskId, count, unit)[0];
+      let latestDataPoint = await getLatestDataPoint(
+        zoneId,
+        deskId,
+        count,
+        unit,
+      )[0];
```

Comment:
co kdyz to bude prazdny list? To spadne

Reply:
Proc se tu vlastne nevola neco aby si dostal jeden ?

Reply:
Pridam check

Reply:
Ten influx client s vysledkama query pracuje defaultne jako s polem. Ale asi by se to dalo nejak prepsat no:
```
queryClient.queryRows(fluxQuery, {
      next: (row, tableMeta) => {
        const tableObject = tableMeta.toObject(row);
        const result: DataPoint = {
          timestamp: new Date(tableObject._time),
          deskId: tableObject.deskId,
          zoneId: tableObject.zoneId,
          value: tableObject._value,
        };
        results.push(result);
      }
```

Reply:
to by bylo asi lepsi:D

Reply:
Hele v dalsim PRku budu mit query, u ktery budu chtit vratit dva data pointy, takze to v tom clientu asi menit nebudu. Kdyby se to zas nekdy rozsypalo, tak to muzu prepsat. Ted merguju 😄